src/components/NavigationBar/NavigationBar.css

```diff
@@ -9,7 +9,6 @@ a:focus {
   top: 0;
   width: 100%;
   box-sizing: border-box;
-  background-color: #fff;
   color: #1a3257;
```

Comment:
Oh I see we removed the background color here. Could you share more details on why you moved it from here to the jsx component?